buildpack.md

```diff
@@ -266,12 +266,12 @@ If a cache is provided, the lifecycle:
 
 Before the next re-build:
 - The lifecycle MUST do both or neither of the following:
-  - Restore Layer Content Metadata to `<layers>/<layer>.toml`, excluding the `launch`, `build`, and `cache` booleans.
+  - Restore Layer Content Metadata to `<layers>/<layer>.toml`, excluding the `[types]` table.
   - Restore layer contents to the `<layers>/<layer>/` directory.
 
 #### Ignored Layers
 
-Layers marked `launch = false`, `build = false`, and `cache = false` behave like temporary directories, available only to the authoring buildpack, existing for the duration of a single build.
+Layers marked `launch = false`, `build = false`, and `cache = false` (or when the `[types]` table does not exist) behave like temporary directories, available only to the authoring buildpack, existing for the duration of a single build.
```

Comment:
I'm not sure if we need to add this parenthesis. The same thoughts are on line 278.

Reply:
Changed as part of https://github.com/buildpacks/spec/pull/209/commits/f2eb2af7d05dff04d6570e2057321c45adca96ac..